hypervisor/Makefile

```diff
@@ -374,6 +374,11 @@ else
 endif
 endif
 
+ifeq ($(FIRMWARE),uefi)
+install: $(HV_OBJDIR)/$(HV_FILE).32.out
+	install -D $(HV_OBJDIR)/$(HV_FILE).32.out $(DESTDIR)/usr/lib/acrn/$(HV_FILE).$(BOARD).efi.32.out
```

Comment:
the output file name contains string of "efi" might cause confusion.
user might be confused with "acrn.efi" which is a efi application.

Reply:
ok just remove efi from file name should be fine right ? file name would look like acrn.apl-nuc.32.out in rpm .